### ISSUE TITLE
Wayland: Seek bar does not seek when clicked

### ISSUE THREAD

**Issue by accumulator** [clementine-player/Clementine#6608]
### Before posting

Please follow the steps below and check the boxes with [x] once you did the step.

- [x] I checked the issue tracker for similar issues
- [ ] I checked the [changelog](https://github.com/clementine-player/Clementine/blob/master/Changelog) if the issue is already resolved
- [ ] I tried the latest Clementine build from [here](https://builds.clementine-player.org/)

### System information

Please provide information about your system and the version of Clementine used.

- Operating System: Debian bullseye
- Clementine version: Both 1.4.0-rc1 and 1.3.1 from buster

### Expected behaviour / actual behaviour

Using Clementine on Plasma/X11 clicking the seek bar (both moodbar enabled and disabled) always worked as expected, e.g. the playback continued from the click location.

Since switching to Plasma/wayland clicking the seek bar almost never results in a seek. Almost, because one in 20-30 clicks it does seem to respond.

### Steps to reproduce the problem (only for bugs)

Play FLAC track. Click on seek bar to another location. Observe you being ignored

**Comment by mtbu** [comment]
I've been having a similar issue on X11 (LXQT/i3), version 1.4.0rc1+495+g10bf5dc17-2 from Arch repos, and then 1.4.0rc1-468-g1a3828e2c built from Git.

Occasionally I could seek but mostly it didn't work.
Playing around a bit more it seemed whenever the progress popup was visible the moodbar / progressbar (tried both).

By disabling the popup and preventing it appearing (commenting out popup_->show() and similar popup set text / etc lines in src/widgets/tracksliderslider.cpp) it seems to work fine now - I click somewhere and it seeks correctly).

**Comment by cyferc** [comment]
Yes Clementine 1.4 rc2 is unusable on Kubuntu 21.04, seek bar is not clickable. Only broken on x11 session but it works fine on Wayland session.
